### PR TITLE
22.4 dockerfile build

### PR DESCRIPTION
**Pull request by lennardstam**
upgrade ubuntu from 18.04 to 22.04.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,16 +1,129 @@
...
+        librsvg2-dev \
+        libprotobuf-dev \
+        libprotobuf-c-dev \
+        libprotobuf-c1 \
+        libxslt1-dev && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN apt-get -y update --fix-missing
+
+RUN git clone --single-branch -b pdok-7-6-4-patch-5 https://github.com/pdok/mapserver/ /usr/local/src/mapserver
```
Comment (lbam) [inline Dockerfile:67]:
```suggestion
RUN git clone --depth=1 --single-branch -b pdok-7-6-4-patch-5 https://github.com/pdok/mapserver/ /usr/local/src/mapserver
```

Takes about a minute less to download (MapServer has a LONG history).

File: Dockerfile [hunk 2]
```diff
@@ -1,16 +1,129 @@
...
+        gcc \   
+        g++ \
+        libfreetype6-dev \
+        libglib2.0-dev \
+        libcairo2-dev \
+        git \
+        locales \
+        make \
+        patch \
+        openssh-server \
```
Comment (lbam) [inline Dockerfile:22]: Why are we installing an SSH server?

Comment (emacgillavry) [reply Dockerfile:22]: We can do without, I suppose. @lennardstam ?

Comment (lennardstam) [reply Dockerfile]: sure, package is removed from the list

**Review by lbam — APPROVED** [review]

**Pull request merged**